subsys/net/ip/net_context.c

```diff
@@ -2166,6 +2180,113 @@ static int set_context_proxy(struct net_context *context,
 #endif
 }
 
+static int set_context_nodelay(struct net_context *context,
+			     const void *value, size_t len)
+{
+	ARG_UNUSED(context);
+	ARG_UNUSED(value);
+	ARG_UNUSED(len);
+	return 0;
+}
+
+static int get_context_nodelay(struct net_context *context,
+			      void *value, size_t *len)
+{
+	ARG_UNUSED(context);
+	ARG_UNUSED(value);
+	ARG_UNUSED(len);
+	return 0;
+}
+
+#if IS_ENABLED(CONFIG_NET_TCP_KEEPALIVE)
+static int set_context_keep_idle(struct net_context *context,
+			     const void *value, size_t len)
+{
+	struct tcp *p = NULL;
+
+	if (len > sizeof(uint32_t)) {
```

Comment:
Shouldn't this rather be `len != sizeof(uint32_t)`? We expect to get exactly an uint32_t value, nothing else.